### PR TITLE
omf_translator - Working stage - performance improvements / using std…

### PR DESCRIPTION
**Pull request by sstefano69**
… logger / statistics improvement  / code review

### CODE REVIEW

File: src/python/foglamp/translators/omf_translator.py [hunk 1]
```diff
@@ -58,19 +30,20 @@
...
 
-# FIXME: we need to [SHOULD] move this to defaults.py! unless this also needs to be read from database
+# FIXME: it will be removed using the DB layer
 _DB_URL = 'postgresql:///foglamp'
 """DB references"""
 
-_LOG_SCREEN = True
-"""Enable/Disable screen messages"""
+_TRACE_EXECUTION = False
+"""Enable/Disable info/debug messages"""
```
Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: Why do we need _TRACE_EXECUTION and method debug_msg_write now if we are using foglamp logger?

File: src/python/foglamp/translators/omf_translator.py [hunk 2]
```diff
@@ -790,7 +740,43 @@ def omf_types_creation():
...
+            _sensor_id = sensor_info
+            _measurement_id = "measurement_" + _sensor_id
+
+            tmp_type = _sensor_name_type[_sensor_id]
+
+            _type_sensor_id = "type_sensor_id_" + _type_id + "_" + tmp_type
+            _type_measurement_id = "type_measurement_" + _type_id + "_" + tmp_type
+
+            debug_msg_write("INFO", "OMF_object_creation ")
+            omf_object_creation()
```
Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: You are setting up global variables _sensor_id, _measurement_id, tmp_type, _type_sensor_id, _type_measurement_id and then retrieving the values in omf_object_creation method, why not pass these as arguments to the method omf_object_creation and avoid global variables?

File: src/python/foglamp/translators/omf_translator.py [hunk 3]
```diff
@@ -944,40 +936,36 @@ def omf_object_creation():
...
+        _logger.exception(message)
         raise Exception(message)
 
 async def _update_statistics():
-    global _num_readings
-    global _num_discarded_readings
+
     await statistics.update_statistics_value('SENT', _num_sent)
-    _num_readings = 0
-    await statistics.update_statistics_value('UNSENT', _num_unsent)
```
Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: You are setting the _num_unsent in send_info_to_omf method, then should this also not go in statistics table?

Comment (sstefano69) [inline src/python/foglamp/translators/omf_translator.py]: this part has changed in the version I'm working on, having only _num_sent.

File: src/python/foglamp/translators/omf_translator.py (hunk 1 above)
Comment (sstefano69) [inline src/python/foglamp/translators/omf_translator.py]: I'm going to remove debug_msg_write and using _logger.debug

File: src/python/foglamp/translators/omf_translator.py (hunk 2 above)
Comment (sstefano69) [inline src/python/foglamp/translators/omf_translator.py]: I'm going to update in this direction, I will close this PR and open a new one that will have an improve error handling.

Comment (vaibhav-ScaleDB) [inline src/python/foglamp/translators/omf_translator.py]: @sstefano69 I prefer you don't open a new PR, and incorporate changes in this one only. This way we can track what has been changed and incorporate missing functionalities incrementally.

**Comment by sstefano69** [comment]
closed in favor of fogl-363 - #105

**Pull request closed without merging**